release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -2532,3 +2532,20 @@ link:https://access.redhat.com/solutions/5551151[{product-title} 4.6.3 container
 
 To upgrade an existing {product-title} 4.6 cluster to this latest release, see
 xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+
+[id="ocp-4-6-4"]
+=== RHBA-2020:4987 - {product-title} 4.6.4 bug fix update
+
+Issued: 2020-11-16
+
+{product-title} release 4.6.3 is now available. The list of container images and bug fixes included in the update are documented in the link:https://access.redhat.com/errata/RHBA-2020:4987[RHBA-2020:4987] advisory. There are no RPM packages for this release.
```

Comment:
s/4.6.3/4.6.4

Reply:
"are documented" -- should this be "is documented" since it refers to "the list"? Or you might be able to clarify by adding "the", such as:

> The list of container images and **_the_** bug fixes included in the update are documented in

Reply:
argh missed one! thanks, good eye! :slightly_smiling_face: 

re: verb agreement, they are two separate things, but examining this sentence closely actually leads me to a different question which might change this somewhat :sweat_smile: